web/client/utils/GeoStoryUtils.js

```diff
@@ -147,14 +147,14 @@ export const testRegex = (title, filterText, regex = RegExp(filterText, "i")) =>
     return filterText ? regex.test(title) : true;
 };
 /**
- * filter resources based on their title and a filter string
+ * filter resources based on their title (name in case of missing title), description and a filter string
  * @param {object[]} resources resources to filter
  * @param {string} filterText string used to generate regex
  * @param {RegExp} [regex] regex used to test input string, default it uses the filterText
  * @return {object[]} filtered resources
  */
 export const filterResources = (resources = [], filterText, regex = RegExp(filterText, "i") ) => {
-    return filter(resources, r => testRegex(r.data && r.data.title, filterText, regex));
+    return filter(resources, r => testRegex(r.data && (r.data.title || r.data.name), filterText, regex) || testRegex(r.data && r.data.description, filterText, regex));
```

Comment:
Would this be fine either?
```suggestion
    return filter(resources, r => testRegex(r.data && (r.data.title || r.data.name || r.data.description), filterText, regex));
```

Reply:
Both are good but aren't the same
1) You search by title/name and if nothing is found you'll search by description
2) You search by title if not present by name if not present by description 
ie {title: "pippo" , description: "aaaaa"}
if you type aa 
1) It finds the element by description
2)It has no result
Just tell me which one you prefer

Reply:
right